<launch>
	<arg name="udp_ip" default="192.168.1.104"/>
	<arg name="udp_port" default="8080"/>
	
    <arg name="chunk_size" default="4000"/>
    
    <arg name="device_id" default="0" />
    <arg name="jpg_quality" default="50" />
    <arg name="fps" default="10" />
    
    <arg name="input_width" default="3840" />
    <arg name="input_height" default="1080" />
    
    <arg name="display_name" default="udpwebcam_pub" /> 
    
    <node name="$(arg display_name)" pkg="udpwebcam" type="udpwebcam_pub.py" output="screen">
        <param name="udp_port" type="int" value="$(arg udp_port)" />
	    <param name="udp_ip" type="string" value="$(arg udp_ip)" />
	    <param name="chunk_size" type="int" value="$(arg chunk_size)" />
	    <param name="device_id" type="int" value="$(arg device_id)" />
        <param name="jpg_quality" type="int" value="$(arg jpg_quality)" />
        <param name="fps" type="int" value="$(arg fps)" />
        <param name="input_width" type="int" value="$(arg input_width)" />
        <param name="input_height" type="int" value="$(arg input_height)" />
    </node>

</launch>

